test/image/compare_pixels_test.js

```diff
@@ -93,11 +108,33 @@ function untestableFilter(mockName) {
     return cond;
 }
 
+/* gl2d mocks that have multiple subplots
+ * can't be generated properly on CircleCI
+ * at the moment.
+ *
+ * For more info see:
+ * https://github.com/plotly/plotly.js/pull/980
+ *
+ */
+function untestableGL2DonCIfilter(mockName) {
+    var cond = [
+        'gl2d_multiple_subplots',
+        'gl2d_simple_inset',
+        'gl2d_stacked_coupled_subplots',
+        'gl2d_stacked_subplots',
+
+        // not sure why this one still fails on CircleCI
+        'gl2d_pointcloud-basic'
```

Comment:
@etpinard I suppose the `pointcloud` CI testing failure is for the reason discussed from this comment https://github.com/plotly/plotly.js/pull/980#issuecomment-251177127

Any way I can simulate the workings of CI *locally*, or read about its constituents? I could try and see what's going on with the `pointcloud` test blending.  Although it should not hold back this PR.